examples/DVCLive-HuggingFace.ipynb

```diff
@@ -1,167 +1,321 @@
 {
- "cells": [
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "!pip install accelerate datasets dvclive evaluate 'transformers[torch]' --upgrade"
-   ]
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3SJ8SY6ldmsS"
+      },
+      "source": [
+        "### How to do Experiment tracking with DVCLive\n",
+        "\n",
+        "What you will learn ?\n",
+        "\n",
+        "- Fine-tuning a model on a binary text classification task\n",
+        "- Track Machine Learning experiments with DVCLive\n",
+        "- Visualize results and create report\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nxiSBytidmsU"
+      },
+      "source": [
+        "#### Git & Install Dependencies\n",
+        "\n",
+        "- Install accelerate , Datasets , evaluate , transformers and dvclive\n",
+        "- Start a Git repo . Your experiments will be saved in a commit but hidden in\n",
+        "  order to not clutter your repo.\n",
+        "- Initialize DVC\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "CLRgy2W4dmsU"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install dvc pandas\n",
+        "!pip install accelerate datasets dvclive evaluate 'transformers[torch]' --upgrade"
```

Comment:
DVC should be installed by dvclive so no need to specify it.
Merged pandas in the second line.

```suggestion
        "!pip install accelerate datasets dvclive evaluate pandas 'transformers[torch]' --upgrade"
```